### DISCUSSION TITLE
On-prem instance with SPNEGO/GSSAPI Kerberos authentication

### DISCUSSION THREAD

**Discussion by aiobofh** [ory/.github#15] [Ideas]
Hi masters!

Is there a way to configure Hydra to authenticate users using a Kerberos token, using SPNEGO/GSSAPI as credentials, achieving single-sign-on in an enterprise environment? Sadly I could not see anything regarding this in the docs. However - Hope still lives on!

This would be such a nice thing <3

Kind regards,
AiO

**Comment by vinckr** [comment]
Hello @aiobofh 

I think
> authenticating users using a Kerberos token, using SPNEGO/GSSAPI as credentials,

would be something that would fit more in the scope of [Ory Kratos](https://github.com/ory/kratos/).

There is currently an effort underway to [implement SAML](https://github.com/ory/kratos/pull/2653). Kerberos and other legacy enterprise protocols have been requested once(?) before, but so far not much interest from the community. 
If you are up to it feel free to open a [Feature Request](https://github.com/ory/kratos/issues/new?assignees=&labels=feat&template=FEATURE-REQUEST.yml) or [Design Document](https://github.com/ory/kratos/issues/new?assignees=&labels=rfc&template=DESIGN-DOC.yml) (the latter if you already have an idea how to implement). 
Thanks 🤗 

edit: if anyone has a way to make this work just with Ory Hydra, please share details in here!
